site/content/docs/5.3/dark-mode.md

```diff
@@ -644,6 +644,223 @@ Additional variables for dark-mode (temporary)
   </nav>
 </div>
 
+### Tags
+
+<h4 class="mt-3">No theme</h4>
+
+<div class="border border-tertiary p-3">
+  <ul class="list-unstyled d-flex gap-2 flex-wrap m-0">
+    <li>
+      <input type="checkbox" class="btn-check" id="btncheck-mobile1" autocomplete="off">
+      <label class="tag" for="btncheck-mobile1"><span class="visually-hidden">Filter by</span>Mobile</label>
+    </li>
+    <li>
+      <input type="checkbox" class="btn-check" id="btncheck-tv1" autocomplete="off" checked>
+      <label class="tag" for="btncheck-tv1">
+        <svg width="1.5rem" height="1.5rem" viewBox="0 0 1000 1000" aria-hidden="true" focusable="false">
+          <path fill="currentColor" d="M75,200V720H225v80H775V720H925V200H75ZM500,755a30,30,0,1,1,30-30A30,30,0,0,1,500,755Zm365-95H135V260H865V660Z"></path>
+        </svg>
+        <span class="visually-hidden">Filter by</span>TV
+      </label>
+    </li>
+    <li><a class="tag" href="#">1. Introduction</a></li>
+    <li>
+      <span class="tag" id="labelTag11">
+        Dismissible tag
+        <button class="close" aria-labelledby="labelTag11"><span class="visually-hidden">Close</span></button>
+      </span>
+    </li>
+    <li><span class="tag tag-sm" id="labelTag21">
+      <svg width="1rem" height="1rem" fill="currentColor" aria-hidden="true" focusable="false">
+        <use xlink:href="/docs/5.3/assets/img/boosted-sprite.svg#buy"/>
```

Comment:
In case we keep this page after 5.3:
```suggestion
        <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
```
And all occurences below.